prowler/lib/utils/utils.py

```diff
@@ -162,3 +163,42 @@ def is_owned_by_root(file_path: str) -> bool:
             f"{file_path}: {e.__class__.__name__}[{e.__traceback__.tb_lineno}]: {e}"
         )
         return False
+
+
+def strip_ansi_codes(s):
+    import re
+
+    ansi_escape = re.compile(r"(?:\x1B[@-_][0-?]*[ -/]*[@-~])")
+    return ansi_escape.sub("", s)
+
+
+def print_boxes(messages: list, report_title: str):
```

Comment:
It'd be great to move this to another file/folder since the `utils` is just a place to put a lot of different things at the end.